library/Rules/Email.php

```diff
@@ -5,6 +5,14 @@ class Email extends AbstractRule
 {
     public function validate($input)
     {
-        return is_string($input) && filter_var($input, FILTER_VALIDATE_EMAIL);
+        if (class_exists('\Egulias\EmailValidator\EmailValidator')) {
+            $strictValidator = new \Egulias\EmailValidator\EmailValidator();
```

Comment:
Actually you can.

The autoloader is just called when you create an instance of the object.